glean/metrics.yaml

```diff
@@ -18,6 +18,28 @@
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
 sample:
+  android_daemon_status:
+    type: event
+    lifetime: ping
+    send_in_pings:
+      - main
+    description: |
+      A status report of the Android Daemon
```

Comment:
Hm, this is a bit vague. What are we trying to answer with this new collection?

Reply:
Done :)